src/dispatchers.unit.test.ts

```
@@ -1,20 +1,14 @@
-    type RequestContext,
-} from '@fedify/fedify';
+import { Activity, type RequestContext } from '@fedify/fedify';
 import {
     likedDispatcher,
     nodeInfoDispatcher,
     outboxDispatcher,
 } from './dispatchers';
 
 import { ACTOR_DEFAULT_HANDLE } from './constants';
-import * as lookupHelpers from './lookup-helpers';
+import { ContextData } from 'app';
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)